### PR TITLE
[ALLUXIO-3240] Use Configuration.getMs instead of Configuration.getLong for TTL

### PR DESCRIPTION
**Pull request by Reidddddd**
https://alluxio.atlassian.net/browse/ALLUXIO-3240

### CODE REVIEW

**Comment by Reidddddd** [comment]
Looks like i did something wrong...

**Comment by Reidddddd** [comment]
any hint? :(

**Comment by aaudiber** [comment]
hm, looks like Jenkins is having transient errors trying to pull from github

**Comment by aaudiber** [comment]
Jenkins, test this please

**Comment by apc999** [comment]
the code change LGTM. Can you address the failing tests.

**Comment by Reidddddd** [comment]
Still trying to figure out why those UTs failed...

**Comment by Reidddddd** [comment]
Hi bin @apc999 , is there a way i can test specific tests on local instead of running all tests which always fails on my local mac...

**Comment by Reidddddd** [comment]
Both FileSystemMasterTest#ttlFileDelete, #setTtlForFileWithNoTtl pass on local "mvn test", i don't understand why they keep failure on Jenkins.

**Comment by Reidddddd** [comment]
ping @apc999, @aaudiber for review

**Comment by apc999** [comment]
@Reidddddd LGTM.  thanks for the improvement

**Comment by Reidddddd** [comment]
ping @aaudiber

File: core/common/src/main/java/alluxio/Constants.java [hunk 1]
```diff
@@ -146,7 +146,7 @@
   public static final int THRIFT_STOP_TIMEOUT_SECONDS = 60;
 
   // Time-to-live
-  public static final long NO_TTL = -1;
+  public static final long NO_TTL = 0;
```
[the remark this answers is not in the document]
Comment (Reidddddd) [reply core/common/src/main/java/alluxio/Constants.java]: Checked. alluxio version i'm using doesn't support -1, but master branch works. 
I'll revert unnecessary changes

File: core/server/master/src/main/java/alluxio/master/file/meta/TtlBucketList.java [hunk 2]
```diff
@@ -46,7 +46,7 @@ public TtlBucketList() {
    * @return the bucket containing the inode, or null if no such bucket exists
    */
   private TtlBucket getBucketContaining(Inode<?> inode) {
-    if (inode.getTtl() == Constants.NO_TTL) {
+    if (inode.getTtl() <= Constants.NO_TTL) {
```
Comment (aaudiber) [inline core/server/master/src/main/java/alluxio/master/file/meta/TtlBucketList.java]: Prefer the original or `inode.getTtl() < 0`.

Comment (Reidddddd) [reply core/server/master/src/main/java/alluxio/master/file/meta/TtlBucketList.java]: what's the different between **inode.getTtl() < 0** and **inode.getTtl() <= -1 (Constants.NO_TTL)**

Comment (aaudiber) [reply core/server/master/src/main/java/alluxio/master/file/meta/TtlBucketList.java]: They are the same today, but using `inode.getTtl() <= Constants.NO_TTL` could lead to a bug later. It adds an additional constraint on `Constants.NO_TTL` that all values representing "no ttl" must be less than or equal to `Constants.NO_TTL`. Previously we could change `Constants.NO_TTL` to `-10` or `MIN_INT` without any consequences.

On further thought, I think it's better to stick with `inode.getTtl() == Constants.NO_TTL` and use `-1` only to mean "no ttl". I don't think it's very helpful to users to be able to use `-5`, and it's just simpler for us to have a single `Constants.NO_TTL` value which means "no ttl".

**Pull request merged**